pom.xml

```diff
@@ -130,6 +130,7 @@
     <okapi.version>0.4.2</okapi.version>
     <hive.group>org.apache.hive</hive.group>
     <hive.classifier>core</hive.classifier>
+    <leveldbjni.group>org.fusesource.leveldbjni</leveldbjni.group>
```

Comment:
I know it's minor, but, I would also not stick it in the middle of the Hive-related settings. Just maybe put it after arrow.version or something.

Reply:
ok, thanks, will do it.